docs/gettingstarted/index.rst

```diff
@@ -210,39 +210,31 @@ Getting started with Flocker
       You should have the Flocker client installed on your laptop and flocker-node installed on some servers: either VMs on your laptop, or real instances on cloud infrastructure.
       Now you can try our simple tutorial: a Python web application and a Redis server.
 
-      .. code-block:: console
+      .. container:: hidden
```

Comment:
There is a comment within this block which says "Create the files to be downloaded with curl, but don't show download links for them"